gluten-ut/spark35/src/test/scala/org/apache/gluten/utils/velox/VeloxTestSettings.scala

```diff
@@ -100,6 +100,7 @@ class VeloxTestSettings extends BackendTestSettings {
     .exclude(
       "Process Infinity, -Infinity, NaN in case insensitive manner" // +inf not supported in folly.
     )
+    .exclude("cast from timestamp II") // Rewrite test for Gluten
```

Comment:
We need a comment to explain why we exclude this test. `Rewrite test for Gluten` semms not a suitable reason. is it work defferently with velox?